linker/Mono.Linker.Steps/MarkStep.cs

```diff
@@ -461,12 +483,12 @@ protected void MarkAssembly (AssemblyDefinition assembly)
 
 			MarkAssemblySpecialCustomAttributes (assembly);
 
-			MarkCustomAttributes (assembly);
+			LazyMarkCustomAttributes (assembly);
```

Comment:
For an assembly this should be done in MarkAssemblySpecialCustomAttributes (and possibly rename the method) as we can skip known attributes there